account_payment_terminal/static/src/js/views/payment_terminal_form/payment_terminal_form_controller.js

```diff
@@ -0,0 +1,160 @@
+odoo.define("account_payment_terminal.OCAPaymentTerminalFormController", function (
+    require
+) {
+    "use strict";
+
+    var rpc = require("web.rpc");
+    var FormController = require("web.FormController");
+    var core = require("web.core");
+
+    var _lt = core._lt;
+
+    var OCAPaymentTerminalFormController = FormController.extend({
+        init: function () {
+            this.transaction_id = null;
+            this.oca_payment_terminal_id = null;
+            this.proxy_url = null;
+            this._super.apply(this, arguments);
+        },
+        _show_error: function (error_message) {
+            this.$el.find("p").text(error_message);
+            this.$el.find("p").css("color", "red");
```

Comment:
Suggestion : remove this line and add css attribute in the first place(error msg)